src/main/java/io/debezium/connector/vitess/VitessSourceInfoStructMaker.java

```diff
@@ -37,11 +34,11 @@ public Schema schema() {
     @Override
     public Struct struct(SourceInfo sourceInfo) {
         final Struct res = super.commonStruct(sourceInfo)
-                .put(SourceInfo.KEYSPACE_NAME_KEY, this.keyspace)
-                .put(SourceInfo.SCHEMA_NAME_KEY, sourceInfo.getTableId().schema())
+                .put(SourceInfo.KEYSPACE_NAME_KEY, sourceInfo.database())
```

Comment:
I think we'd better update Vitess's `SourceInfo.database()` to empty string and add `SourceInfo.getKeyspace()` and use it here for consistency

Reply:
FYI @gunnarmorling @sonne5, chiming in here since this is already merged -- just spoke to @shichao-an. This isn't blocking and we'll get it in after the release.

Reply:
No worries. That said, we had an infra issue today, which stopped us from doing the CR1 release as planned, and we'll try again tomorrow (Friday). This is to so, if you wanted to bring in this change, we could try.

If so, feel free to create a PR, but I'd leave it to @jpechane to merge (or not) it before the release, because I am not sure where he is in the process, and whether adding more at this point will throw a wrench between the gears.

Reply:
This is the PR to address comment https://github.com/debezium/debezium-connector-vitess/pull/60

@jeffchao to clarify, I think the PR ^ is just some cosmetic changes, I think the changes that make a differences are:
- https://github.com/debezium/debezium-connector-vitess/pull/58 - which adds `keyspace` in source
- https://github.com/debezium/debezium-connector-vitess/pull/59 - which remove `schema` and `db` in source
they will catch the CR1 release already

Reply:
Thanks for addressing this. The API change was settled in #59 so we can use it in CR1. The proposed fix #60 is good to have but not required for release.